src/algorithms/core.ts

```diff
@@ -36,6 +37,11 @@ export interface AlgorithmInput {
    * @see [MapCanvasProjection](https://developers.google.com/maps/documentation/javascript/reference/overlay-view#MapCanvasProjection)
    */
   mapCanvasProjection: google.maps.MapCanvasProjection;
+
+  /**
+   * The `forceRecalculate` option requires cluster recalculation.
```

Comment:
Is "requires" the right word.

Also may be you can expand on when `forceRecalculate` forces re-clusrtering?